src/Driver/driver_configs.jl

```diff
@@ -154,12 +156,12 @@ function Atmos_GCM_Configuration(
                                             polynomialorder=N,
                                             meshwarp=meshwarp)
 
-    return DriverConfiguration(name, N, FT, array_type, solver_type, model,
-                               mpicomm, grid, numfluxnondiff, numfluxdiff,
-                               gradnumflux)
+    return DriverConfiguration(AtmosGCMConfigType(), name, N, FT, array_type,
+                               solver_type, model, mpicomm, grid,
+                               numfluxnondiff, numfluxdiff, gradnumflux)
 end
 
-function Ocean_BoxGCM_Configuration(
+function OceanBoxGCMConfiguration(
```

Comment:
`examples/OceanBoxGCM/homogenous_box.jl` and `ocean_gyre.jl` need to updated to match this, right?

Reply:
Oops.